### PR TITLE
Update C# Sample sample applications for 2021. June verification

### PR DESCRIPTION
**Pull request by sgchoi5**
Updated the github link of TV/Maps
Removed outdated C# sample apps - Sample Telephony
Removed invalid information (All native samples having "(Tutorial)" in the application title have been removed recently)

### CODE REVIEW

**Comment by sgchoi5** [comment]
L.G.T.M

File: docs/application/dotnet/samples/appfw/overview.md [hunk 1]
```diff
@@ -173,10 +173,6 @@ for TD:
 			<ul>
 				<li><a href="/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.Messages.MessagePort.html" target="_blank">Class MessagePort</a></li>
 			</ul></p>
-			<p>In addition, there is similar native sample application.<br>
-			<ul>
-				<li><a href="/development/sample/native/AppFW/%28Tutorial%29_Message_Port" target="_blank">Native version</a></li>
-			</ul></p>
 			</td>
 		</tr>
 		<tr>
```
Comment (nawab-samsung) [inline docs/application/dotnet/samples/appfw/overview.md:178]: Line 187 can we use the following as an image?
NoUI

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@sgchoi5 I have a suggestion, If you like please incorporate the same, rest all the changes are fine.

File: docs/application/dotnet/samples/appfw/overview.md (hunk 1 above)
Comment (sgchoi5) [reply docs/application/dotnet/samples/appfw/overview.md:178]: I've changed No UI only to Image file.

Comment (nawab-samsung) [reply docs/application/dotnet/samples/appfw/overview.md:178]: @sgchoi5 I could not get you.
image
I was just asking about this.

Comment (sgchoi5) [reply docs/application/dotnet/samples/appfw/overview.md:178]: @nawab-samsung How about this ?
Image file with black background color with text like

No UI Available => Service application - No UI Available    or
                               Service Application - No UI Available

Comment (nawab-samsung) [reply docs/application/dotnet/samples/appfw/overview.md:178]: That's fine, great thanks for applying this.

Comment (nawab-samsung) [reply docs/application/dotnet/samples/appfw/overview.md:178]: @sgchoi5 sorry but the spelling is incorrect in round one 

Serivce--> Service 

Please, can you let me know if the position of the circle UI image is fine. 
SharedScreenshot

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@sgchoi5 Sorry, but a few doubts.

**Comment by sgchoi5** [comment]
Sorry, fixed typo and layout issue

**Comment by sgchoi5** [comment]
@nawab-samsung Is it OK? I can't check the results, because https://docs1.stg.tizen.org/staging/1455 is not updated anymore.

**Comment by nawab-samsung** [comment]
@sgchoi5 Thanks a lot, Staging is down for a couple of days , even I am not able to view the results.

But the latest changes/commits are fine. And I think after checking it, I will approve.

**Review by nawab-samsung — APPROVED** [review]
All the updated links work as on Git, not able to verify on staging as staging seems to be down.
@sgchoi5 Thanks for implementing my suggestion.

**Comment by sgchoi5** [comment]
Will fix it asap in master branch

**Pull request merged**